group_vars/all/main.yml

```
@@ -181,3 +181,8 @@ container_gid: 0
 # directly.
 default_storage_class_rwx: "manual"
 default_storage_class_rwo: "local-path"
+
+# Ingress class name
+# A default ingress class name is not necessarily defined in a k8s cluster.
+# We can specify which one to use by defining the ingressClassName in the Ingress spec
+default_ingress_class_name: "nginx"
```

A carriage return is missing:

```suggestion
default_ingress_class_name: "nginx"

```